client/state/stored-cards/actions.js

```diff
@@ -13,10 +13,17 @@ import {
 import 'calypso/state/stored-cards/init';
 
 export const addStoredCard = ( cardData ) => ( dispatch ) => {
-	return wp
-		.undocumented()
-		.me()
-		.storedCardAdd( cardData.token, cardData.additionalData )
+	return wp.req
```

Comment:
Note: apart from unit tests, this thunk is unused by anything, but I thought it wise to keep it as part of the set just in case.